presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -1035,6 +1035,41 @@ public void testSelectAllFromTable()
         assertQuery("SELECT y FROM (SELECT r.* AS (x, y, z) FROM region r ORDER BY y DESC LIMIT 2)", "VALUES 'MIDDLE EAST', 'EUROPE'");
     }
 
+    @Test
+    public void testSelectAllFromOuterScopeTable()
```

Comment:
Do we have any tests where the alias shadows an alias or table name from an outer scope? Maybe also one-level removed, where the alias binds against the outer scope, but there's another alias with the same name one level further?